Localization/en-US.hjson

```diff
@@ -22,56 +16,86 @@ Mods: {
 		SwitchToNormal: Switch to Normal Mode
 		SwitchToExpert: Switch to Expert Mode
 		SwitchToMaster: Switch to Master Mode
-		DifficultyCommandUsage: Usage: /Difficulty <0(Normal)|1(Expert)|2(Master)>
+		DifficultyCommandUsage: Usage: /difficulty <0(Normal)|1(Expert)|2(Master)>
 
+		Items: {
+			FakeDemonHeart: {
+				DisplayName: Fake Demon Heart
+				Tooltip: ""
+			}
```

Comment:
The one in BossExpertise.Items.FakeDemonHeart not BossExpertise.ItemName.FakeDemonHeart. Which is empty so we need to move it into the correct location. It is simpler if you do the localization stuff so it stays mergeable since if i want to make changes to it now I have include your changes to solve merge conflicts.

Reply:
Alright, committing lang changes